src/views/admin/adminusers.hbs

```diff
@@ -0,0 +1,213 @@
+		   
```

Comment:
This file should be named add-users.hbs based on this folder structure https://github.com/at-internship/at-sso-ui/blob/develop/src/views/admin/user/add-user.hbs

Reply:
Please do not mix behaviors of:
index.hbs
add-user.hbs

Each view has its own purpose.